### PR TITLE
Use the generator script

### PR DESCRIPTION
**Pull request by mallardduck**
This is just a PR I'm opening to show how simply blade-ui-kit/blade-icons#144 could help maintain icon sets.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,9 +10,15 @@
             "homepage": "https://driesvints.com"
         }
     ],
+    "repositories": [
```
Comment (mallardduck) [inline composer.json:13]: Note - I'll update the PR after the new feature/script is merged in at: https://github.com/blade-ui-kit/blade-icons/pull/144

File: generate [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+GenerateCommandBuilder::create('blade-heroicons')
+    ->fromSourceSvgDirectory('/dist/optimized')
+    ->withIconSets([
+        IconSetConfig::create('outline')
+            ->setOutputFilePrefix('o-'),
+        IconSetConfig::create('solid')
+            ->setOutputFilePrefix('s-'),
+    ])
+    ->useSingleIconSet()
+    ->run();
```
Comment (driesvints) [inline generate:18]: Move this within the `./bin` directory.

File: composer.json [hunk 3]
```diff
@@ -29,16 +39,15 @@
...
-    "config": {
-        "sort-packages": true
-    },
+    ],
     "minimum-stability": "dev",
-    "prefer-stable": true
+    "prefer-stable": true,
+    "scripts": {
+        "generate": "./bin/generate"
+    }
```
Comment (driesvints) [inline composer.json:52]: Can you undo the re-sorting of keys here?

Comment (mallardduck) [reply composer.json:52]: yeah, sorry I applied composer normalize after adding the scripts.

**Comment by driesvints** [comment]
@mallardduck sorry, realized you might want to revert the changes I made to the composer.json file in order to continue.

**Comment by mallardduck** [comment]
@driesvints - NP at all. my setup works for the relative paths too.

**Comment by driesvints** [comment]
Thanks @mallardduck!

**Pull request merged**